opentelemetry-sdk/src/metrics/meter_provider.rs

```diff
@@ -244,14 +245,37 @@ impl MeterProviderBuilder {
     }
 
     /// Associates a [MetricReader] with a [MeterProvider].
+    /// [`with_periodic_exporter`] can be used to add a PeriodicReader which is
+    /// the most common use case.
     ///
-    /// By default, if this option is not used, the [MeterProvider] will perform no
-    /// operations; no data will be exported without a [MetricReader].
+    /// A [MeterProvider] will perform no operations without [MetricReader]
+    /// added.
     pub fn with_reader<T: MetricReader>(mut self, reader: T) -> Self {
         self.readers.push(Box::new(reader));
         self
     }
 
+    /// Adds a [`PushMetricExporter`] to the [`MeterProvider`] and configures it
+    /// to export metrics at **fixed** intervals (60 seconds) using a
+    /// [`PeriodicReader`].
+    ///
+    /// To customize the export interval, set the
+    /// **"OTEL_METRIC_EXPORT_INTERVAL"** environment variable (in
+    /// milliseconds).
+    ///
+    /// Most users should use this method to attach an exporter. Advanced users
+    /// who need finer control over the export process can use
+    /// [`PeriodicReaderBuilder`] to configure a custom reader and attach it
+    /// using [`with_reader`].
+    pub fn with_periodic_exporter<T>(mut self, exporter: T) -> Self
```

Comment:
it is intentional to pick with_periodic_exporter and not with_periodic_reader. This is what was already followed in Logs/Traces (with_batch_exporter, not with_batch_processor).
Given this is a convenience API, better to avoid "reader" which is irrelevant to most users.